main.py

```diff
@@ -369,3 +370,20 @@ def enable_lldp(self):
         msg_error = "Some interfaces couldn't be found and activated: "
         return jsonify({msg_error:
                         error_list}), 400
+
+    @rest('v1/time', methods=['GET'])
+    def get_time(self):
+        """Get LLDP polling time."""
+        return jsonify({"Polling time in seconds": self.polling_time}), 200
+
+    @rest('v1/time/<time_sec>', methods=['POST'])
```

Comment:
I think you should not put the value of what you want to change in the URL, but use the payload for that.
I also am not sure if POST is the best method here.

Reply:
Thanks for the suggestion @ajoaoff , I changed this method to use the payload. Sorry for the long delay to answer.